### PR TITLE
Remove balance data from Redux and rely on Tanstack

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Review by mvaivre — APPROVED** [review]
LGTM! Only one tiny nit, so approving.

**Comment by nop33** [comment]
Where's the nit? 😆

**Review by mvaivre — APPROVED** [review]
LOL. My comment wasn't sent and wasn't cached... and I can't remember what it was... SO... LGTM I guess 😆

**Comment by nop33** [comment]
HAHAHAHAHA

File: apps/desktop-wallet/src/modals/SendModals/useAreAmountsWithinAvailableBalance.ts
Comment (mvaivre) [inline apps/desktop-wallet/src/modals/SendModals/useAreAmountsWithinAvailableBalance.ts:1]: Actually I remember! My nit was that I'm not super convinced by this hook file floating in the SendModals folder :)

Comment (nop33) [reply apps/desktop-wallet/src/modals/SendModals/useAreAmountsWithinAvailableBalance.ts:1]: Ah haha! Well, that's the only place it is needed. Only in the send workflow. Optimally we should have a `src/features/send/` folder and put it there. But that might require quite some refactoring which I don't think it's priority now. WDYT?

Comment (mvaivre) [reply apps/desktop-wallet/src/modals/SendModals/useAreAmountsWithinAvailableBalance.ts:1]: Ok!

**Pull request merged**